onboard/catkin_ws/src/motion_planning/scripts/task_utils.py

```diff
@@ -79,6 +84,42 @@ def at_pose(current_pose, desired_pose, linear_tol=0.1, angular_tol=3):
     return linear and angular
 
 
+def at_vel(current_twist, desired_twist, linear_tol=0.1, angular_tol=0.3):
+    """Check if within tolerance of a twist (linear and angular velocity)
+    
+    """
+    lin_curr_vel_vec = np.array([current_twist.linear.x, current_twist.linear.y, current_twist.linear.z])
+    lin_curr_vel_mag = np.linalg.norm(lin_curr_vel_vec)
+    lin_des_vel_vec = np.array([desired_twist.linear.x, desired_twist.linear.y, desired_twist.linear.z])
+    lin_des_vel_mag = np.linalg.norm(lin_des_vel_vec)
+    linear = math.fabs(lin_curr_vel_mag - lin_des_vel_mag) < linear_tol
+
+    ang_curr_vel_vec = np.array([current_twist.angular.x, current_twist.angular.y, current_twist.angular.z])
+    ang_curr_vel_mag = np.linalg.norm(ang_curr_vel_vec)
+    ang_des_vel_vec = np.array([desired_twist.angular.x, desired_twist.angular.y, desired_twist.angular.z])
+    ang_des_vel_mag = np.linalg.norm(ang_des_vel_vec)
+    angular = math.fabs(ang_curr_vel_mag - ang_des_vel_mag) < angular_tol
```

Comment:
Once the changes are made to shorten the `norm` calls, the number of lines will be about the same, and I think since it's so short, not having to think through how another function is called might be clearer.